package/Makefile

```diff
@@ -15,7 +15,6 @@ ROOT_DIR := $(realpath $(WHEREAMI)/../)
 include $(ROOT_DIR)/make/system-id.mk
 
 TOOLS_DIR := $(ROOT_DIR)/tools
-BUILD_DIR := $(ROOT_DIR)/build
```

Comment:
It is defined on line 20 which shows as original so I guess it was double defined